Date: Jun 23 18:59:49 2014

From: 5fee2e840b33c83e5b72ebfc59438fede4900fd2 Jun 29 19:48:28 2014

Subject: [PATCH] RDKSYSINTSW-948: Moved iamrbus implementation from fusiondale to dbus

	Added rdk dbus configuration so wrote a bbappend to apply this
	as a patch to system.conf.

	Modified iarmbus, iarmmgrs recipe to include dbus as a dependency.

	Modified rdkbrowser recipe to link libdbus-1.so through a patch.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: rra <ridish.ra@lnttechservices.com>
---

diff -urN dbus-1.6.18.orig/bus/system.conf.in dbus-1.6.18/bus/system.conf.in
--- dbus-1.6.18.orig/bus/system.conf.in	2014-06-23 17:19:37.479688002 +0000
+++ dbus-1.6.18/bus/system.conf.in	2014-06-23 17:20:02.159688723 +0000
@@ -47,8 +47,8 @@
 
     <!-- Holes must be punched in service configuration files for
          name ownership and sending method calls -->
-    <deny own="*"/>
-    <deny send_type="method_call"/>
+    <allow own="*"/>
+    <allow send_type="method_call"/>
 
     <!-- Signals and reply messages (method returns, errors) are allowed
          by default -->
